Tests/BowTests/Transformers/OptionTTest.swift

```diff
@@ -40,4 +40,32 @@ class OptionTTest: XCTestCase {
     func testFunctorFilterLaws() {
         FunctorFilterLaws<OptionTPartial<ForId>>.check(generator: self.generator)
     }
+
+    func testToLeftWithSome() {
+        property("toLeft for .some should build a correct EitherT") <- forAll { (a: Int, b: String) in
+            let optionT = OptionT<ForId, Int>.fromOption(.some(a))
+            return optionT.toLeft({ b }) == EitherT<ForId, Int, String>.left(a)
```

Comment:
With `@autoclosure` you don't need the braces in `{ b }`.

Reply:
Do we prefer the @autoclosure syntax? I mean, in case we want to use a function as a parameter we are going to need `()`.